webapp/src/main/java/org/apache/falcon/resource/SchedulableEntityManager.java

```diff
@@ -57,6 +59,91 @@ public APIResult getStatus(@Dimension("entityType") @PathParam("type") String ty
         }
     }
 
+    /**
+     * Delete the specified entity.
+     * @param request Servlet Request
+     * @param type Valid options are cluster, feed or process.
+     * @param entity Name of the entity.
+     * @param ignore colo is ignored
+     * @return Results of the delete operation.
+     */
+    @DELETE
+    @Path("delete/{type}/{entity}")
+    @Produces({MediaType.TEXT_XML, MediaType.TEXT_PLAIN, MediaType.APPLICATION_JSON})
+    @Monitored(event = "delete")
+    @Override
+    public APIResult delete(
+            @Context HttpServletRequest request, @Dimension("entityType") @PathParam("type") final String type,
+            @Dimension("entityName") @PathParam("entity") final String entity,
+            @Dimension("colo") @QueryParam("colo") String ignore) {
+        throw FalconWebException.newAPIException("delete on server is not" +
```

Comment:
Hi ,

I have confirmed its working fine in embedded mode it checks intoSchedulableEntityManageProxy class not in SchedulableEntityManager :

$  bin/falcon entity -type process -name DWH-CI-daily-supply5 -delete
falcon/default/DWH-CI-daily-supply5(process) removed successfully (KILLED in WF_ENGINE)